cmd/bootstrap/cmd/qc.go

```diff
@@ -20,7 +20,13 @@ func constructRootQC(block *flow.Block, votes []*model.Vote, allNodes, internalN
 		log.Fatal().Err(err).Msg("failed to generate QC participant data")
 	}
 
-	qc, err := run.GenerateRootQC(block, votes, participantData, identities)
+	qc, invalidVotesErr, err := run.GenerateRootQC(block, votes, participantData, identities)
+	if len(invalidVotesErr) > 0 {
+		for _, err := range invalidVotesErr {
+			log.Warn().Err(err).Msg("invalid vote")
+		}
+	}
```

Comment:
Since we do not intend to fail when invalid votes are used, could we move this logic to `GenerateRootQC`? If move the logic down, we won't need to update the function signature and return multiple types of errors. Since we are logging them immediately, we could log them individually after evaluating each vote.

Reply:
I tried, we could log them inside GenerateRootQC, but the logger is not the same logger as being used by the bootstrap cmd. Therefore, in ansible script, it won't print that log if there is invalid vote, which might seem as if there is no invalid vote. 

To be safe, I decided to return invalid votes as errors, and use the same logger to print log and invalid votes, it will guarantee that the log for invalid votes will be printed along with other info logs.